actions/chains/bgp_prefix_exceeded_chain.yaml

```diff
@@ -0,0 +1,75 @@
+---
+  chain:
+      -
+        name: "show_bgp_neighbours"
```

Comment:
Her majesty would have my head if I spelt it any other way (except where I've cut and pasted in the other actions ;) ). I too have had this battle most of my career and am thankful that I can abbreviate it on most occasions as 'nei' and not have to wear out my backspace key to delete the u.